docs/user_guide.md

```diff
@@ -17,18 +17,18 @@ cd Simoorg
 ```
 When installing from the source, we recommend that you run all the unit and integration tests that are included with the source code, before starting the installations. You can do this by running the command
 ```
-    python setuptools.py test
+    sudo python setup.py test
```

Comment:
Can you explain what issues you faced or error you got when you ran the previous command?